src/controller/order-controller.ts

```diff
@@ -147,8 +152,8 @@ async function captureOrderHandler(
 ) {
   const { orderID } = request.body as { orderID: string };
 
-  const { data, httpStatus } = await captureOrder(orderID);
-
+  const responseData = await captureOrder(orderID);
```

Comment:
This should support passing in headers too. Ex:
```suggestion
  const responseData = await captureOrder({ body: { orderID }, headers });
```

Reply:
inside `captureOrder `, we have `captureOrderAPI`  which takes the headers.